### PR TITLE
migrating security check to rules_docker

### PR DESCRIPTION
**Pull request by nlopezgi**
Continuing migration of all rules needed to build deterministic base containers to rules_docker repo.
This PR migrates a security_check that is still EXPERIMENTAL.
Security check requires gcloud to properly run.
This PR is only migrating existing code, will improve documentation (particularly as related to requirements and expectations of the checker) in upcoming PR

NOTE: PR is not yet passing CI, will need to resolve issue related to permissions and re-run.

### CODE REVIEW

**Comment by smukherj1** [comment]
Looks like you need to fix a couple of buildifier issues
```
 buildifier: found 1 format issue in your WORKSPACE, BUILD and *.bzl files
Please download buildifier 0.22.0 and run the following command in your workspace:

buildifier tests/docker/security/BUILD
:bazel: buildifier: found 1 lint issue in your WORKSPACE, BUILD and *.bzl files
docker/security/security_check.bzl:15: module-docstring: The file has no module docstring.
```

File: docker/security/security_check.bzl [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Copyright 2017 Google Inc. All rights reserved.
```
Comment (xingao267) [inline docker/security/security_check.bzl]: license header

File: docker/security/security_check.py [hunk 2]
```diff
@@ -0,0 +1,330 @@
+"""Checks the specified image for security vulnerabilities."""
```
Comment (xingao267) [inline docker/security/security_check.py]: license

File: docker/security/security_check.bzl (hunk 1 above)
Comment (nlopezgi) [reply docker/security/security_check.bzl]: done

File: docker/security/security_check.py (hunk 2 above)
Comment (nlopezgi) [reply docker/security/security_check.py]: done

**Pull request merged**